### PR TITLE
Adds necesary files for PyPi release

### PR DESCRIPTION
**Pull request by lcortesg**

### CODE REVIEW

**Comment by lcortesg** [comment]
[https://pypi.org/project/EEGraSP/](Package released in PyPi)

File: eegrasp/tests/compare_distance_method.py
Comment (aweinstein) [inline eegrasp/tests/compare_distance_method.py:1]: Is this file necessary for the PyPi release?

File: eegrasp/tests/compare_with_pygsp.py
Comment (aweinstein) [inline eegrasp/tests/compare_with_pygsp.py:1]: Is this file necessary for the PyPi release?

File: eegrasp/tests/df_100.pkl
Comment (aweinstein) [inline eegrasp/tests/df_100.pkl:1]: Is this file necessary for the PyPi release?

File: eegrasp/tests/df_pos.pkl
Comment (aweinstein) [inline eegrasp/tests/df_pos.pkl:1]: Is this file necessary for the PyPi release?

File: pyproject.toml
Comment (aweinstein) [inline pyproject.toml:1]: Use alejandro.weinstein@gmail.com for my email.
Change "Herera" to "Herrera".
Camilo's email is wrong.

(I don't know why I can't comment on specific lines.)

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        'Bug Tracker': 'https://github.com/gsp-eeg/eegrasp/issues',
+        #'Try It Online': 'https://mybinder.org/v2/gh/epfl-lts2/eegrasp/master?urlpath=lab/tree/examples/playground.ipynb',
+    },
+    packages=[
+        'eegrasp',
+        'eegrasp.tests',
+    ],
+    package_data={'eegrasp': ['data/pointclouds/*.mat']},
+    test_suite='eegrasp.tests.suite',
+    python_requires='>=3.7',
```
Comment (aweinstein) [inline setup.py:28]: I think we mentioned 3.11 in other places.

File: eegrasp/tests/compare_distance_method.py
Comment (lcortesg) [reply eegrasp/tests/compare_distance_method.py:1]: Agreed, files were put in their original location

File: eegrasp/tests/compare_with_pygsp.py
Comment (lcortesg) [reply eegrasp/tests/compare_with_pygsp.py:1]: Agreed, files were put in their original location

File: eegrasp/tests/df_100.pkl
Comment (lcortesg) [reply eegrasp/tests/df_100.pkl:1]: Agreed, files were put in their original location

File: eegrasp/tests/df_pos.pkl
Comment (lcortesg) [reply eegrasp/tests/df_pos.pkl:1]: Agreed, files were put in their original location

File: setup.py (hunk 1 above)
Comment (lcortesg) [reply setup.py:28]: Fixed

**Review by aweinstein — APPROVED** [review]
